Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "prio"
-version = "0.16.7"
+version = "0.17.0-alpha.0"
```

Comment:
Don't we want to change this version number after branching? Otherwise, the new release branch will have a 0.17 version number.

Reply:
I already created the release/0.16 branch with the then-current tip of main, so it won't be affected.